VARIABLES.md

```diff
@@ -67,6 +67,7 @@
 | s3_inventory_format | Output format for S3 inventory results. Can be Parquet, ORC, CSV | string | `ORC` | no |
 | s3_inventory_update_schedule | Cron schedule to update S3 inventory tables (if enabled). Defaults to every 12 hours. | string | `0 */12 * * *` | no |
 | s3_lifecycle_policy_transition_period | Number of days for transition to a different storage class using lifecycle policy. | string  | `30` | no |
+| s3_object_expiration_days | Number of days after which objects in apiary managed schema buckets expire | number | `null` | no
```

Comment:
Since you removed the variable from `variables.tf`, you need to remove this line too.